### PR TITLE
[WIP] Symbiflow packages

### PR DESCRIPTION
**Pull request by PiotrZierhoffer**
This adds Symbiflow-specific vtr, yosys and yosys-plugins.

This should be merged after https://github.com/litex-hub/litex-conda-eda/pull/23

### CODE REVIEW

**Comment by PiotrZierhoffer** [comment]
This will not go green, due to dependencies

**Comment by PiotrZierhoffer** [comment]
Not yet sure why didn't Travis start here

File: .travis.yml [hunk 1]
```diff
@@ -118,6 +146,21 @@ jobs:
     dist: xenial
     env:
     - PACKAGE=pnr/vtr
+  - stage: "EDA Tools - PnR"
```
Comment (mithro) [inline .travis.yml:149]: The `vtr-XXX` packages should be in a separate pull request?

File: .travis.yml [hunk 2]
```diff
@@ -143,6 +186,21 @@ jobs:
     osx_image: xcode8.3
     env:
     - PACKAGE=pnr/nextpnr/generic
+  - stage: "EDA Tools - PnR"
```
Comment (mithro) [inline .travis.yml:189]: The Mac support for vtr should be a separate pull request, right?

File: .travis.yml [hunk 3]
```diff
@@ -118,6 +146,21 @@ jobs:
     dist: xenial
     env:
     - PACKAGE=pnr/vtr
+  - stage: "EDA Tools - PnR"
+    os: linux
+    dist: xenial
+    env:
+    - PACKAGE=pnr/vtr-gui
+  - stage: "EDA Tools - PnR"
```
Comment (mithro) [inline .travis.yml:154]: Can we separate out symbiflow synthesis stuff from the symbiflow-vtr stuff into separate pull requests?

**Comment by mithro** [comment]
Does this need me to review?

**Comment by ajelinski** [comment]
`symbiflow-vtr`, `symbiflow-vtr-gui`, `symbiflow-yosys` and `symbiflow-yosys-plugins` have been all added through other separate PRs.
